libpod/define/pod_exit_policy.go

```diff
@@ -0,0 +1,34 @@
+package define
+
+import "fmt"
+
+// PodExitPolicies includes the supported pod exit policies.
+var PodExitPolicies = []string{"continue", "stop"}
```

Comment:
do not duplicate the strings please, use `string(PodExitPolicyContinue)` and `string(PodExitPolicyStop)`